docs/tutorials.rst

```diff
@@ -0,0 +1,467 @@
+
+            Simply add the ``?`` at the end of any module or function to see a detailed description of imports or
+            expected inputs. Try it out on ``nwb2bids?`` to see the library's welcome message.
+
+            The **nwb2bids** library is also designed for easy auto-completion in interactive environments like
+            IPython or Jupyter notebooks; try it out on by hitting your tab button after typing ``nwb2bids.testing.`` to
+            see all the functions exposed to that submodule.
+
+Now we should have a directory that looks something like:
+
+.. code-block:: text
+
+    ephys_tutorial_dataset/
+    ├── ephys_session_3.nwb
+    ├── DO_NOT_CONVERT.nwb
+    └── some_sessions/
+        ├── ephys_session_1.nwb
+        └── ephys_session_1.nwb
+
+Now, to convert only the subset of files under `some_sessions/` to BIDS, we can run the following command:
+
+.. tabs::
+    .. tab:: CLI
+
+        .. code-block:: bash
+
+            nwb2bids convert ~/.nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions ^
+                --bids-directory ~/.nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions_bids_dataset
+
+    .. tab:: Python Library
+
+        .. code-block:: python
+
+            import pathlib
+
+            import nwb2bids
+
+            nwb_paths = [pathlib.Path.home() / ".nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions"]
+            bids_directory = pathlib.Path.home() / ".nwb2bids/tutorials/ephys_tutorial_dataset/some_sessions_bids_dataset"
+
+            nwb2bids.convert_nwb_dataset(nwb_paths=nwb_paths, bids_directory=bids_directory)
+
+And our BIDS dataset should look like:
+
+.. code-block:: text
+
+    ephys_tutorial_dataset/some_sessions_bids_dataset/
+    ├── dataset_description.json
+    ├── participants.tsv
+    ├── participants.json
+    └── sub-subject+1/
+        ├── sub-subject+1_sessions.tsv
+        ├── sub-subject+1_sessions.json
+        ├── ses-session+1/
+        │   └── ecephys/
+        │       ├── sub-subject+1_ses-session+1_ecephys.nwb
+        │       ├── sub-subject+1_ses-session+1_channels.tsv
+        │       ├── sub-subject+1_ses-session+1_channels.json
+        │       ├── sub-subject+1_ses-session+1_electrodes.tsv
+        │       ├── sub-subject+1_ses-session+1_electrodes.json
+        │       ├── sub-subject+1_ses-session+1_probes.tsv
+        │       └── sub-subject+1_ses-session+1_probes.json
+        └── ses-session+2/
+            └── ecephys/
+                ├── sub-subject+1_ses-session+2_ecephys.nwb
+                ├── sub-subject+1_ses-session+2_channels.tsv
+                ├── sub-subject+1_ses-session+2_channels.json
+                ├── sub-subject+1_ses-session+2_electrodes.tsv
+                ├── sub-subject+1_ses-session+2_electrodes.json
+                ├── sub-subject+1_ses-session+2_probes.tsv
+                └── sub-subject+1_ses-session+2_probes.json
```

Comment:
Yes, a future PR will add `.json` 'bubbling' up to the top level here and the docs will then be updated accordingly

Reply:
- #67